lib/object_storage/miq_ftp_storage.rb

```diff
@@ -0,0 +1,102 @@
+require 'util/miq_ftp_lib'
+require 'util/miq_object_storage'
```

Comment:
These still say "util/". but that's not where these live now.

Reply:
I had thought about fixing this, but forgot when I saw the tests were passing, at first, I was confused at why this was still working, but I think the answer is simple:

The corresponding PR for this in `manageiq-gems-pending` is no merged, so without running this with `manageiq-cross_repo_tests`, this still loads just fine and the specs pass.

Will look into fixing these next week.  Thanks!

Reply:
Can you do a cross_repo test with the removal PR in the manageiq-gems-pending?  Then this would prove that out.